app/views/transport/transportMeans/active/AddAnotherBorderTransportView.scala.html

```diff
@@ -0,0 +1,69 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.transport.transportMeans.active.routes
+@import models.LocalReferenceNumber
+@import viewModels.components.InputYesNoViewModel._
+@import config.FrontendAppConfig
+@import views.html.components.{Button, ErrorSummary, InputYesNo, ListWithActions}
+@import views.html.components.Heading
+@import viewModels.transport.transportMeans.active.AddAnotherBorderTransportViewModel
+
+@this(
+    mainTemplate: MainTemplate,
+    formHelper: FormWithCSRF,
+        heading: Heading,
+        listWithActions: ListWithActions,
+    inputYesNo: InputYesNo,
+    errorSummary: ErrorSummary,
+    button: Button
+)
+
+@(form: Form[Boolean], lrn: LocalReferenceNumber, mode: Mode, viewModel: AddAnotherBorderTransportViewModel)(implicit request: Request[_], messages: Messages, config: FrontendAppConfig)
+
+@mainTemplate(
+    title = viewModel.title,
+    showBackLink = true,
+    lrn = Some(lrn)
+) {
+
+    @errorSummary(form.errors)
+
+    @heading(
+        heading = viewModel.heading,
+        caption = Some(messages("transport.activeBorderMeansOfTransport.caption"))
+    )
+
+    @listWithActions(viewModel.listItems)
+
+    @formHelper(action = viewModel.onSubmitCall, Symbol("autoComplete") -> "off") {
+
+        @if(viewModel.allowMoreActiveBorderTransports) {
+            @inputYesNo(
+                field = form("value"),
+                yesNoType = YesNoWithLegendAndHint(
```

Comment:
`YesNoWithLegendAndHint` can be deleted. inputYesNo takes a hint as one of the parameters